measures/measures-schema.yaml

```diff
@@ -82,6 +82,9 @@ definitions:
       eMeasureId:
         description: Identifier for Electronic Clinical Quality Measures (ECQM).
         type: ['null', string]
+      eMeasureUuid:
```

Comment:
Ahhh, I misunderstood how fields are optional unless defined explicitly - will make this a string-only type and leave it optional.